src/main/kotlin/io/github/mojira/arisa/modules/PrivacyModule.kt

```diff
@@ -0,0 +1,54 @@
+package io.github.mojira.arisa.modules
+
+import arrow.core.Either
+import arrow.core.extensions.fx
+import io.github.mojira.arisa.domain.Issue
+import java.time.Instant
+
+class PrivacyModule : Module {
+    private val patterns: List<Regex> = listOf(
+        """\(Session ID is token:""".toRegex(),
+        """\S+@\S+\.\S{2,}""".toRegex()
+    )
+
+    override fun invoke(issue: Issue, lastRun: Instant): Either<ModuleError, ModuleResponse> = with(issue) {
+        Either.fx {
+            assertNull(securityLevel).bind()
+
+            var string = ""
+
+            if (created.isAfter(lastRun)) {
+                string += "$summary $environment $description "
+            }
```

Comment:
It might make sense to also scan changelog entries created after last run, and also resolve if fromString does not succeed, but toString does.